### PR TITLE
Fix printing tests

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: test/cdmparse.jl [hunk 1]
```diff
@@ -27,7 +27,7 @@ end
 
 @testset "CDM parsing invalid directory" begin
     baddir = joinpath(RTS_GMLC_DIR, "../../test")
-    @test_throws SystemError PowerSystems.csv2ps_dict(baddir, 100.0)
+    @test_throws ErrorException PowerSystems.csv2ps_dict(baddir, 100.0)
```
Comment (claytonpbarrows) [inline test/cdmparse.jl]: This throws a `SystemError` on my machine. Am I doing something wrong?

Comment (daniel-thom) [reply test/cdmparse.jl]: It throws ErrorException on mine!  I thought this was a merge problem.  I'll investigate.

Comment (daniel-thom) [reply test/cdmparse.jl]: I pushed a fix.

**Comment by jd-lara** [comment]
This looks good, once we merge this and #207 the testing errors in `dev` should be minimal.

**Pull request merged**